nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/Candidate.java

```diff
@@ -887,17 +893,26 @@ private boolean isValidStatusChange(
     return allowedOperations.contains(attemptedOperation);
   }
 
-  /**
-   * This checks which operations this organization is allowed to perform on this candidate.
-   *
-   * <p>This requires retrieving metadata about each creator's organization, which is not persisted
-   * in the database yet. Once we persist the necessary metadata, we can simplify this check. TODO:
-   * Persist full organization hierarchy for creators and simplify this validation.
-   */
-  public Set<CandidateOperation> getAllowedOperations(
+  private void validateCreators(
```

Comment:
Can we extract the validations to their own class?

Reply:
Done :)